### PR TITLE
What: Updating project resource to match contributor strategy format

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Why:

* contributor strategy uses `<!-- TODO: -->` to indicate action

This change addresses the need by:

* inserting  where the project is required to make changes.

### CODE REVIEW

File: project-resources/templates/embargo-policy.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ replacement of you from this list or the Security Contacts.
 
 ## Disclosure Timeline
 
-This project sustains a **$X day disclosure timeline** to ensure we provide a
+This project sustains a <!-- TODO: **$X day disclosure timeline** --> to ensure we provide a
```
Comment (lumjjb) [inline project-resources/templates/embargo-policy.md:27]:
```suggestion
This project sustains a **<!-- TODO: $X day  --> disclosure timeline** to ensure we provide a
```

**Review by lumjjb — APPROVED** [review]
couple nits, lgtm

File: project-resources/README.md [hunk 2]
```diff
@@ -73,4 +77,17 @@ and discussions as guidance when determining the content of their updates.
 
 It is highly recommended that you seek peer review for your updates beyond that
 of the Technical Leads and Chairs. More information on contributions to this
-repo may be found in the [contributing file](../CONTRIBUTING.md)
+repo may be found in the [contributing file](../CONTRIBUTING.md).
```
Comment (lumjjb) [inline project-resources/README.md:80]: actually this link is now supposedly https://github.com/cncf/tag-security/blob/main/CONTRIBUTING/README.md 
```suggestion
repo may be found in the [contributing file](../CONTRIBUTING.md).
```

Comment (TheFoxAtWork) [reply project-resources/README.md:80]:
```suggestion
repo may be found in the [contributing file](../CONTRIBUTING/README.md).
```

**Pull request merged**